src/bun.js/webcore/blob.zig

```diff
@@ -2410,51 +2410,84 @@ pub const Blob = struct {
         // If the optional end parameter is not used as a parameter when making this call, let relativeEnd be size.
         var relativeEnd: i64 = @intCast(i64, this.size);
 
+        if (args.ptr[0].isString()) {
```

Comment:
needs a 0 length check, `blob.slice()` will crash

Reply:
callframe.arguments(3) presets everything to to `JSValue.zero` and .ptr avoids the bounds check

Reply:
`isString()` will crash if jsvalue is zero

Reply:
`blob.slice(number)` also has the same problem. maybe we should have a check for zero in `isString()`

Reply:
fixed